bootcamp/RAG/zilliz_bento_RAG.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>Since this is a demo about Zilliz Cloud + BentoCloud. Let's replace all usage of Milvus to Zilliz Cloud (except for function names such as MilvusClient()).</p><p> </p><p>E.g.,</p><p> </p><p>In this demo, we show how to use open-source embedding model and LLM on BentoCloud with vector database on Zilliz Cloud to build a RAG application. </p><p> </p><p> </p><p>BentoCloud offers serverless infrastructure tailored for model inference, operating in conjunction with BentoML, an open-source model serving framework, to facilitate the easy creation and deployment of high-performance model services. Zilliz Cloud is fully-managed service for open-source vector database - Milvus, with flexible pricing and ease of management. You can sign-up for free on BentoCloud and Zilliz Cloud. Later we will use the API keys from both services to finish the demo.</p><p> </p><p>On BentoCloud, you can setup model services in <code style="font-weight: 400; letter-spacing: 0.01em;">Deployments</code>, and the corresponding END_POINT and API are located in <code style="font-weight: 400; letter-spacing: 0.01em;">Playground</code> -&gt; <code style="font-weight: 400; letter-spacing: 0.01em;">Python</code>. On Zilliz Cloud, we can get endpoint and token in <code style="font-weight: 400; letter-spacing: 0.01em;">Cluster Details</code>.</p><p> </p><p>After we get API credentials from both platform and the corresponding endpoint, we can start to do RAG! </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
No problem, fixed